vmdk_plugin/utils/refcount/refcnt.go

```diff
@@ -241,6 +241,10 @@ func (r RefCountsMap) Decr(vol string) (uint, error) {
 	}
 
 	rc.count--
+
+	if rc.count < 0 {
+		log.Warning("Decr: refcnt is negative, name=%s refcnt=%d", vol, rc.count)
```

Comment:
So the corrupted slot will stay in the map ? 
I think it would be better to simply change to ` if rc.count <=0` (and update message) on line 234. This will also delete the map entry.